src/as_execution/common.rs

```diff
@@ -22,6 +23,17 @@ pub(crate) fn call_module(
         Ok(v) => v,
         Err(_) => abi_bail!("negative amount of coins in Call"),
     };
+    if function.len() > usize::from(MAX_FUNCTION_NAME_LENGTH) {
+        abi_bail!("Function name length too high");
+    }
+    // safe to unwrap as MAX_PARAMETERS_SIZE is const value manually defined
+    if param.len()
+        > usize::try_from(MAX_PARAMETERS_SIZE)
+            .expect("Cannot convert MAX_PARAMETERS_SIZE to usize")
+    {
+        abi_bail!("Parameters length too high");
+    }
+
     let env = get_env(ctx)?;
     let bytecode = env.get_interface().init_call(address, raw_coins)?;
```

Comment:
idea: add "function" and optional "parameters" fields to init_call, and perform the check on Massa side to avoid duplicating the config parameters